petshop-be/web/src/main/java/com/turkninja/petshop/v1/UserResource.java

```diff
@@ -34,6 +34,20 @@
         return new ResponseEntity<>(HttpStatus.CREATED);
     }
 
+    @PostMapping("/register")
```

Comment:
Do you have any other suggestions for endpoint?

Reply:
it may be register-admin